packages/services/f-feature-management/README.md

```diff
@@ -1,19 +1,32 @@
 # Feature Management 
 
-**This is a WIP**
-
-This service allows querying of feature flags as provided by our backend Feature Management service.
+This service allows querying of feature flags as configured using the Feature Management service.
 
 ## Usage
 
+**This service is a WIP. The instructions cover the current way of working. This is subject to change.**
+
 To instantiate the SDK:
 
 ```javascript
-import featureManagement from '@justeat/f-feature-management';
+import createFeatureManagementInstance from '@justeat/f-feature-management';
+
+const configJson = //string containing configuration json.  The remote load functionality has not yet been added to this lib.
+
+const contextGetter = function(){
+  return {
+    country: 'uk',
+    anonUserId: getCookieValue('je-auser-id') //getCookieValue() implementation to be provided by integrator
+  };
+}
```

Comment:
You can update this example to use an arrow function that fits a little better with the style of code used in the web projects.

```suggestion
const contextGetter = () => ({
  country: 'uk',
  anonUserId: getCookieValue('je-auser-id') // getCookieValue() implementation to be provided by integrator
})
```